README.md

```diff
@@ -41,3 +41,6 @@ Our backend is split into 2 sections: our API app & our database. Our API is wri
 
 
 ## Frontend
+1. The frontend is written in React.[If you need help with react, refer to its ](https://reactjs.org/docs/getting-started.html "documentation")
```

Comment:
Can you fix the "documentation" at the end.  It seems like it is not showing correctly in the rich text view and the sentence ends after "refer to its..."

Reply:
Hello, I just fixed it!